rampwf/tests/kits/titanic/submissions/starting_kit/submission.py

```diff
@@ -0,0 +1,22 @@
+from sklearn.pipeline import Pipeline
+from sklearn.pipeline import make_pipeline
+from sklearn.compose import ColumnTransformer
+from sklearn.impute import SimpleImputer
+from sklearn.linear_model import LogisticRegression
+
+
+def get_pipeline():
+    numeric_cols = ['Pclass', 'Age', 'SibSp', 'Parch',
+                    'Fare']
+    to_drop = ['Name', 'Sex', 'Ticket', 'Cabin', 'Embarked']
```

Comment:
as before, if they are all remaining columns just use the default `remainder='drop'` in the columntransformer